src/input_xml.F90

```diff
@@ -1482,6 +1607,7 @@ subroutine read_geometry_xml()
       deallocate(coeffs)
 
       ! Boundary conditions
+      s % bc = BC_TRANSMIT
```

Comment:
This line `s % bc = BC_TRANSMIT`  was added by Nick in a commit "Set default surface boundary condition to TRANSMIT". I think it is redundant too and will revert it.